fwup.conf

```diff
@@ -215,6 +219,8 @@ task upgrade.a {
     on-resource bcm2708-rpi-b-plus.dtb { fat_write(${BOOT_PART_OFFSET}, "bcm2708-rpi-b-plus.dtb.new") }
     on-resource w1-gpio-pullup.dtbo { fat_write(${BOOT_PART_OFFSET}, "overlays/w1-gpio-pullup.dtbo.new") }
     on-resource dwc2.dtbo { fat_write(${BOOT_PART_OFFSET}, "overlays/dwc2.dtbo.new") }
+    on-resource pi3-disable-bt.dtbo { fat_write(${BOOT_PART_OFFSET}, "overlays/pi3-disable-bt.dtbo.new") }
+    on-resource pi3-disable-bt.dtbo { fat_write(${BOOT_PART_OFFSET}, "overlays/pi3-disable-bt.dtbo.new") }
```

Comment:
This looks like a duplicate to the previous line. I'm not sure if that's a problem.